signac/testing.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2019 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
```

Comment:
Let's add a module doc-string briefly describing what should go into this module.